incubator/hydra/values.yaml

```diff
@@ -37,6 +40,13 @@ config:
   idTokenLifespan: "1h"
   authorizeCodeLifespan: "10m"
   httpsAllowTerminationFrom: 0.0.0.0/0
+  postgresql:
```

Comment:
Hydra supports multiple backing databases. This locks you in with PostgreSQL.

I would do "databaseURL" instead.